coeus-impl/src/main/java/org/kuali/coeus/propdev/impl/core/ProposalDocumentSummaryView.java

```diff
@@ -0,0 +1,46 @@
+package org.kuali.coeus.propdev.impl.core;
+
+import org.apache.commons.lang.StringUtils;
+import org.kuali.rice.kew.api.KewApiServiceLocator;
+import org.kuali.rice.kew.api.doctype.DocumentType;
+import org.kuali.rice.krad.datadictionary.DocumentEntry;
+import org.kuali.rice.krad.document.DocumentRequestAuthorizationCache;
+import org.kuali.rice.krad.document.DocumentViewAuthorizerBase;
+import org.kuali.rice.krad.document.DocumentViewPresentationControllerBase;
+import org.kuali.rice.krad.service.KRADServiceLocatorWeb;
+import org.kuali.rice.krad.uif.UifConstants;
+import org.kuali.rice.krad.uif.view.FormView;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+
+/**
+ * View for a ProposalDocument that is used as a summary view on the action list, used to workaround one
+ * view per document type limitation.
+ */
+public class ProposalDocumentSummaryView extends ProposalDocumentView{
+
+    public ProposalDocumentSummaryView () {
+   		super();
+
+        setRequestAuthorizationCacheClass(DocumentRequestAuthorizationCache.class);
+   	}
+
+    @Override
+    public void performInitialization(Object model) {
+        super.performInitialization(model);
+
+        setObjectPathToConcreteClassMapping(new HashMap<String, Class<?>>());
+        getObjectPathToConcreteClassMapping().put("document", ProposalDevelopmentDocument.class);
+
+        // Workaround for non-set additionalScriptFiles files
+        setAdditionalScriptFiles(new ArrayList<String>());
+    }
+
+    protected DocumentEntry getDocumentEntryForView() {
+        DocumentEntry documentEntry = KRADServiceLocatorWeb.getDocumentDictionaryService().getDocumentEntryByClass(
+                ProposalDevelopmentDocument.class);
+
+        return documentEntry;
+    }
+}
```

Comment:
This was broken by the AuthorizationCache stuff and some assumptions made there, so I had to add this workaround class to get around this.  The root problem and one we ran into when creating this view the first time is that you cant have two DocumentViews for the same document, this gets around that.